### PR TITLE
Non orthogonality

### PR DESCRIPTION
**Pull request by iamkundan**
Comments and Vector Graphics added by Team 4

### CODE REVIEW

File: doc/pages/meshParameters.md [hunk 1]
```diff
@@ -197,3 +197,116 @@ where \f$M\f$ is the maximum magnitude of the following quantities:
...
+### Algorithm for Non-Orthogonality ComputationInterior Face 
+
+Code first for for whether the face is interier face, if Yes! then get it's getter for the neibouring face. then it call for getter for the centre of mass of the owner cell, Face Area vector
+
+\f[
+d = C_n - C_o
+\f]
+
+\f[
+S_f = getAreaVector()
```
Comment (brunoramoa) [inline doc/pages/meshParameters.md:236]: Use vector syntax...

File: doc/pages/meshParameters.md [hunk 2]
```diff
@@ -197,3 +197,116 @@ where \f$M\f$ is the maximum magnitude of the following quantities:
...
+Calculation of Unit Face normal vector:
+
+\f[
+nf = Sf/mag(Sf)
+\f]
+
+Calculation for the non-orthogonality angle in Radian:
+
+\f[
+theta = acos(d_n.nf/[|d_n|.|n|])
```
Comment (brunoramoa) [inline doc/pages/meshParameters.md:299]: use latex syntax in here...

File: doc/Images/nonOnthogonality/fcOT.svg [hunk 3]
```diff
@@ -0,0 +1,510 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (brunoramoa) [inline doc/Images/nonOnthogonality/fcOT.svg:1]: Images, must have vector syntax...

**Review by brunoramoa — CHANGES_REQUESTED** [review]

**Review by iamkundan — COMMENTED** [review]
Okay

File: doc/Images/nonOnthogonality/fcOT.svg (hunk 3 above)
Comment (jayeshgolhar7) [reply doc/Images/nonOnthogonality/fcOT.svg:1]: Images are changed to vector format.

File: doc/pages/meshParameters.md [hunk 4]
```diff
@@ -197,3 +197,117 @@ where \f$M\f$ is the maximum magnitude of the following quantities:
...
+
+### Non-Orthogonality Computation – Interior Faces
+
+There are two faces, Owner face and Neighbou faces as shown in the fig 
+
+\image html iFaceOT.svg "Figure 1: Interior Faces" width=350px 
+
+Where,
+
+\f$\vec{d}\f$ = Vector from Cell Owner Centre of Mass to Neighbour Cell Centre of Mass
```
Comment (jayeshgolhar7) [inline doc/pages/meshParameters.md:220]: wrote the equations in vector format.

Comment (iamkundan) [reply doc/pages/meshParameters.md:220]: completed

File: doc/Images/nonOnthogonality/fcOT.svg (hunk 3 above)
Comment (iamkundan) [reply doc/Images/nonOnthogonality/fcOT.svg:1]: completed

File: doc/pages/meshParameters.md (hunk 4 above)
Comment (brunoramoa) [reply doc/pages/meshParameters.md:220]: They are still not correct...

File: doc/pages/meshParameters.md [hunk 5]
```diff
@@ -197,3 +197,117 @@ where \f$M\f$ is the maximum magnitude of the following quantities:
...
+\f]
+
+\f[
+ \vec{S_f} = getAreaVector()
+\f]
+
+Calculation of Unit Face normal vector:
+
+\f[
+\vec{n_f} =  \vec{S_f}/mag( \vec{S_f})
```
Comment (brunoramoa) [inline doc/pages/meshParameters.md:242]: use latex fraction syntax

File: doc/pages/meshParameters.md [hunk 6]
```diff
@@ -197,3 +197,117 @@ where \f$M\f$ is the maximum magnitude of the following quantities:
...
+\f]
+
+Calculation for the non-orthogonality angle in Radian:
+
+
+\f[
+θ = a cos ( \frac{\ d.\vec{n_f}\ }{|d|.|n|} )
```
Comment (brunoramoa) [inline doc/pages/meshParameters.md:249]: use latex syntax

**Review by brunoramoa — CHANGES_REQUESTED** [review]

**Pull request closed without merging**